Math/NumberTheory/GaussianIntegers.hs

```diff
@@ -149,6 +151,13 @@ primes = [ g
                          in [x :+ y, y :+ x]
          ]
 
+-- |An infinite list of Gaussian primes. See 'primes'.
+orderedPrimes :: [GaussianInteger]
+orderedPrimes = (1 :+ 1): merge l r
+  where (leftPrimes, rightPrimes) = partition (\p -> p `mod` 4 == 3) (tail Sieve.primes)
+        l = [p :+ 0 | p <- leftPrimes]
+        r = [g | p <- rightPrimes, let x :+ y = findPrime p, g <- [x :+ y, y :+ x]]
```

Comment:
Looks like something still goes wrong:

```haskell
> map norm $ take 2 $ drop 75 orderedPrimes
[389,361]
```

Could you please write a test for `orderedPrimes`?

Reply:
While I'm adding tests, are there any other desirable tests other than checking ordering?